regression/cbmc/pragma_cprover_enable1/main.c

```diff
@@ -0,0 +1,16 @@
+int main()
+{
+  int x;
+  int y[1];
+
+#pragma CPROVER check push
+#pragma CPROVER check enable "bounds"
+#pragma CPROVER check enable "signed-overflow"
+  // generate assertions for the following statement
+  x = x + y[1];
+  // pop all annotations
```

Comment:
```suggestion
  // pop all checks added since the last push
```

[nit] also in other test files

Reply:
+1

Reply:
comment removed